_drafts/bridging-digital-and-physical.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout:       post
+title:        Bridging digital and physical with a MakerBot Replicator 2
+author:       Dave Mills
+summary:
+image:        http://image.shutterstock.com/display_pic_with_logo/1451378/316560056/stock-vector-pumpkin-dressed-up-in-stripy-suit-furry-art-character-halloween-greeting-design-316560056.jpg
```

Comment:
I can't get into shutterstock, and maybe we should change that picture
anyway because it's no longer halloween.

On Fri, Apr 8, 2016 at 10:58 AM, Gabby Losch <notifications@github.com>
wrote:

> In _drafts/bridging-digital-and-physical.markdown
> <https://github.com/wework/wework.github.io/pull/8#discussion_r59037730>:
>
> > @@ -0,0 +1,140 @@
> > +---
> > +layout:       post
> > +title:        Bridging digital and physical with a MakerBot Replicator 2
> > +author:       Dave Mills
> > +summary:
> > +image:        http://image.shutterstock.com/display_pic_with_logo/1451378/316560056/stock-vector-pumpkin-dressed-up-in-stripy-suit-furry-art-character-halloween-greeting-design-316560056.jpg
>
> I think it's fine to leave the other images stored in the assets folder,
> since we own that. @i8ramin <https://github.com/i8ramin> -- any reason
> they should be uploaded directly to Cloudinary instead of storing them
> within the app assets?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wework/wework.github.io/pull/8/files/0ac08defe50e329ee4323f18bdd70a0d380e5c46#r59037730>
>